src/OpenTelemetry.Api/Trace/Decision.cs

```diff
@@ -53,11 +53,13 @@ public Decision(bool isSampled, IEnumerable<KeyValuePair<string, object>> attrib
         /// Gets a value indicating whether Span was sampled or not.
         /// The value is not suppose to change over time and can be cached.
         /// </summary>
-        public bool IsSampled { get; private set; }
+        public bool IsSampled { get; }
 
         /// <summary>
         /// Gets a map of attributes associated with the sampling decision.
         /// </summary>
-        public IEnumerable<KeyValuePair<string, object>> Attributes { get; private set; }
+        public IEnumerable<KeyValuePair<string, object>> Attributes { get; }
+
+        // TODO ==
```

Comment:
why do we need comparison operator?

Reply:
it's recommended to override == and != on structs to compare value types